### DISCUSSION TITLE
Create a discord server for this. | 이를 위해 불화 서버를 만드십시오.

### DISCUSSION THREAD

**Discussion by Elektron-blip** [kiki7000/discord.py-components#24] [Ideas]
I think it would be a good idea to start a discord server for this. 
나는 이것을 위해 불화 서버를 시작하는 것이 좋은 생각이라고 생각합니다.
I used google translate for this. Don't judge me. | 나는 이것을 위해 Google 번역을 사용했습니다. 나를 판단하지 마십시오.

**Comment by kiki7000** [comment]
Ok!

**Comment by kiki7000** [comment]
https://discord.gg/pKM6stqPxS
